### ISSUE TITLE
hi, your work is great, on android it worked perfectly as a expected, but on ios, i cannot access the environment variable in the info.plist file 

### ISSUE THREAD

**Issue by quanngovan98** [ByneappLLC/flutter_config#1]
i've add this script 

```
${SRCROOT}/.symlinks/plugins/flutter_config/ios/Classes/BuildXCConfig.rb ${SRCROOT}/.. ${SRCROOT}/Flutter/tmp.xcconfig
echo ".env.staging" > /tmp/envfile
```

**Comment by quanngovan98** [comment]
here my .env file 
![image](https://user-images.githubusercontent.com/39608754/69814703-59edc280-1227-11ea-9342-80b3b716713a.png)
and here how i access it in the info.plist
![image](https://user-images.githubusercontent.com/39608754/69814729-68d47500-1227-11ea-876c-f4b49086e726.png)

**Comment by edTheGuy00** [comment]
Hi there! Did you follow the [iOS setup instructions](https://github.com/ByneappLLC/flutter_config/blob/master/doc/IOS.md) 

This is what works for me, but honestly I haven't tested the iOS side extensively since iOS is not my strongest skill. I'll look into your issue by tomorrow when I have time. Thanks for reporting, I want to make this a robust library.

**Comment by vanquanngo** [comment]
yeah, this is the greatest library for config in flutter, better than any present, i hope that you'll continue to develop it

**Comment by edTheGuy00** [comment]
Hi, You have to add two separate pre-action scripts, and copying the `.env` file has to be the first action. The end result should look something like this: 

I will update the `README` accordingly 

<img width="850" alt="Screen Shot 2563-01-09 at 15 49 22" src="https://user-images.githubusercontent.com/16095735/72052373-a54fc200-32f7-11ea-8a58-a0dd2b542b79.png">

**Comment by edTheGuy00** [comment]
#3

**Comment by nmshaikh** [comment]
Hi there,

Firstly, I want to thank you for the great work you're doing! I'm a Flutter newbie and after an extensive search on how to maintain different build environments, I came across your project and it works like a charm. It has really helped to make my life easier.

I actually also faced the same issue as @quanngovan98 where the tmp.xcconfig file was not being generated and the environment variables were not being read despite following your instructions. I managed to solve the issue by setting the "Provide build settings from" option to my target (Runner). 

<img width="858" alt="pic5" src="https://user-images.githubusercontent.com/28905117/74910061-9713bc00-53f4-11ea-8feb-e3b242f04479.png">

I'm submitting a pull request with an updated README that includes the above steps. Hope this helps!
